Hometask_4_2/Hometask_4_2.py

```diff
@@ -0,0 +1,51 @@
+import random
+import string
+
+
+# Create the list dictionaries from 2 to 10.
+def create_dicts():
+    output = []  # Dicts list
+    for _ in range(random.randint(2, 10)):  # Dictionary counter
+        dictionary = {}
+        for _ in range(random.randint(1, 26)):  # Elements counter
+            # Key(random letter), value(random number from 0 to 100)
+            dictionary[random.choice(string.ascii_lowercase)] = random.randint(0, 100)
+        output.append(dictionary)  # Append every dictionary to the list
+    return output
+
+
+all_values = create_dicts()
+
+
+def all_values_dict():
+    output = {}
+    c = 1
+    for cntr in all_values:
+        for key, value in cntr.items():
+            if key not in output:  # Check whether value exists previously
+                output[key] = {str(c): value}
+            else:
+                output[key][str(c)] = value
+        c += 1
+    return output
+
+
+all_values_dict = all_values_dict()
```

Comment:
Please create file "dict_processor.py" in the root folder of your project.
Import all functions from "Hometask_4_2.py" file to this "dict_processor.py".
And call required functions inside of "dict_processor.py".

So "Hometask_4_2.py" will contain only function definitions.